### PR TITLE
224 prep

### PR DESCRIPTION
**Pull request by tacaswell**
## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: doc/users/github_stats_2.2.3.rst [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+* David Stansby
+* DietmarSchwertberger
+* Elizabeth Seiver
+* Elliott Sales de Andrade
+* Eric Firing
+* Federico Ariza
+* fredrik-1
+* hannah
+* Importance of Being Ernest
+* ImportanceOfBeingErnest
```
Comment (QuLogic) [inline doc/users/github_stats_2.2.3.rst]: It looks like I need to update the `.mailmap` again.

**Comment by tacaswell** [comment]
Proposed 'short' release notes:

-------

REL: v2.2.4 LTS

The forth bug-fix release of the v2.2 LTS series.


 - Fix a memory leak with PySide2
 - rename :math: sphinx directive to :mathmpl: to work with sphinx >
   1.8.  Both versions are now available if sphinx < 1.8
 - avoids numpy deprecation warnings
 - fixes to tests to work with pytest >= 3.8


jquery-ui is no longer bundled in git repo and is downloaded during
the install or sdist process.  If you are installing from pypi the
files will be included, but if you are installing from git you will
need internet access the first time you run `pip install .`.  The
install process tries to cache the files and will not need internet
access on future installations.

**Comment by QuLogic** [comment]
> fixes to tests to work with pytest >= 3.8

Minimum supported version was also bumped to 3.6.

File: tutorials/introductory/customizing.py [hunk 2]
```diff
@@ -184,6 +184,6 @@
 #
 #
 # .. _matplotlibrc: http://matplotlib.org/users/customizing.html
-# .. _ggplot: http://ggplot2.org/
+# .. _ggplot: https://www.anaconda.com/distribution/
```
Comment (ivanov) [inline tutorials/introductory/customizing.py]: woops - https://ggplot2.tidyverse.org/

**Pull request merged**